CMakeLists.txt

```diff
@@ -52,7 +52,11 @@ find_package(Foundation QUIET)
 find_package(SQLite3 REQUIRED)
 
 # Enable `package` modifier for the whole package.
-add_compile_options("$<$<COMPILE_LANGUAGE:Swift>:-package-name;SwiftPM>")
+if(CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
+  add_compile_options("$<$<COMPILE_LANGUAGE:Swift>:-package-name;SwiftPM>" -L/usr/local/lib)
```

Comment:
nit: may be worth a comment that the /usr vs /usr/local separation is a formal FreeBSD policy for all ports not part of the base system, otherwise this difference may look suspicious to future readers.